### ISSUE TITLE
NTR superior semicircular canal dehiscence

### ISSUE THREAD

**Issue by cerivs** [obophenotype/mammalian-phenotype-ontology#2313]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

**superior semicircular canal dehiscence**

>2. Synonyms (e.g., absent spleen)
related 

**superior semicircular canal dehiscence  syndrome
canal dehiscence syndrome
superior canal syndrome
third mobile window syndrome
minor’s syndrome
SSCD**

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

**The absence of bone overlying the superior semicircular canal facing toward the dura of the middle cranial fossa.** 
https://rarediseases.org/rare-diseases/superior-semicircular-canal-dehiscence/

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

**PMID: 28503164
PMID: 9525507** 

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

- 
 **Morphological abnormality of the semicircular canal  http://purl.obolibrary.org/obo/HP_0011380**

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

**ORCID:0000-0002-2244-7917
ZFIN:CVS**
&
**ORCID: 0000-0003-3311-7320
ZFIN:KS  @kevinschaper**

**Comment by sbello** [comment]
Definition of middle cranial fossa - http://teachmeanatomy.info/head/areas/cranial-fossa/middle/ 
I would not call this an abnormality of the semicircular canal, instead I think this is an abnormality of the middle cranial fossa. I think I would put this under abnormal cranium morphology (MP:0000438) or possibly abnormal temporal bone morphology (MP:0005272) as this seems to be the bone that is usually affected.

**Comment by cerivs** [comment]
From the images in the references it looked like the semicircular canals were missing bone but I started out with this as a child of skull morphology.

**Comment by sbello** [comment]
Based on https://www.ncbi.nlm.nih.gov/pubmed/28503164 I've made this a child of both abnormal superior semicircular canal morphology and abnormal temporal bone morphology, as this is a thinning/absence of temporal bone in the section containing the semicircular canals. The MP terms for semicircular canal include both the bone and the interior of the canal. Based on UBERON (UBERON:0007833) the osseus semicircular canal is part of the 'petrous part of temporal bone'

Also, I just noticed that you had an HP term as a parent, did you mean to submit this to the HPO tracker?

MP:0020461 - superior semicircular canal dehiscence
